doc/workflow.md

```diff
@@ -0,0 +1,66 @@
+# Workflow
+
+#### **kube-spawn-dir**
+`/var/lib/kube-spawn` by default, can be set with `--dir`
+
+#### **cluster-dir**
+`<kube-spawn-dir>/default` by default, can be set with `--cluster-name`
+
+## `create`
+
+* _check provided config_ (not implemented)
+* generate a config from defaults in `pkg/config/defaults.go`
```

Comment:
* generate a new configuration for the specified cluster with the built-in defaults (see `pkg/config/defaults.go`)